spark/sql-20/build.gradle

```diff
@@ -156,6 +156,15 @@ dependencies {
     itestImplementation("org.apache.spark:spark-streaming_${project.ext.scalaMajorVersion}:$sparkVersion") {
         exclude group: 'org.apache.hadoop'
     }
+
+    sources(project(":elasticsearch-hadoop-mr"))
+}
+
+// Export generated Java code from the genjavadoc compiler plugin
+artifacts {
+    sourceElements(project.file("$buildDir/generated/java")) {
```

Comment:
does adding the `sourceElements` here conflict with plugin addition ?  ( I don't know if it is additive or a replacement)

Reply:
This isn't _adding_ a configuration. It's adding an artifact to the existing `sourceElements` configuration.

Reply:
Not sure what you mean by this question. Do you mean the scala compiler plugin or the build plugin?